### DISCUSSION TITLE
Where to set number of generated rows?

### DISCUSSION THREAD

**Discussion by veevatodd** [benkeen/generatedata#805] [Q&A] [answered]
First time using the redesigned site and I'm feeling very lost trying to find something simple but critical...the number of records I want to generate in a run. The "tours" seem to indicate this is set in the Preview tab, but all I can see there is a dropdown that goes to 20...I need to generate hundreds. Where does one set this?
![numberofrows](https://user-images.githubusercontent.com/122135924/211127345-609ceea9-d0c3-42b3-ad19-886eb9425877.png)

**Comment by veevatodd** [comment, marked as answer]
I found my answer...the Generate button at the bottom...I totally missed it! Duh...

**Reply by benkeen** [reply]
haha no no, it's interesting to see how people think - perhaps lots of other people have also thought this. Thanks for posting!
